core/src/main/resources/hudson/PluginManager/advanced.jelly

```diff
@@ -66,7 +66,7 @@ THE SOFTWARE.
                 </f:block>
               </f:form>
             </l:hasPermission>
-            <l:hasPermission permission="${app.pluginManager.CONFIGURE_UPDATECENTER}">
+            <l:hasPermission permission="${app.ADMINISTER}">
```

Comment:
None of these permission checks matter anymore, they should just be removed (see new line 30).